### PR TITLE
more updates to the dev tools and more sections

### PR DESCRIPTION
**Pull request by masterfung**
updated the images, added more images and wrote more for the tools in the Develop section.

### CODE REVIEW

File: docs/develop/mainnet/auditing.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security Audits
+
+Security audit is a comprehensive evaluation of the security and stability of a smart contract or decentralized application (dApp) built on the EVM chains. The purpose of this audit is to identify potential vulnerabilities and weaknesses in the code that could compromise the integrity of the dApp, as well as to ensure that the code is robust and free of bugs that could lead to unintended consequences.
```
Comment (danburck) [inline docs/develop/mainnet/auditing.md:3]:
```suggestion
Security audit is a comprehensive evaluation of the security and stability of a smart contract or decentralized application (dApp) built on Evmos. The purpose of this audit is to identify potential vulnerabilities and weaknesses in the code that could compromise the integrity of the dApp, as well as to ensure that the code is robust and free of bugs that could lead to unintended consequences.
```

File: docs/develop/mainnet/auditing.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Security Audits
```
Comment (danburck) [inline docs/develop/mainnet/auditing.md:1]: Consider moving this to testnet, as audits should be performed before moving to mainnet

**Review by danburck — APPROVED** [review]
nice @masterfung left some nit comments

File: docs/develop/mainnet/auditing.md (hunk 2 above)
Comment (masterfung) [reply docs/develop/mainnet/auditing.md:1]: moved

**Pull request merged**